gazebo_ros2_control/src/gazebo_ros2_control_plugin.cpp

```diff
@@ -304,6 +304,11 @@ void GazeboRosControlPlugin::Load(gazebo::physics::ModelPtr parent, sdf::Element
     resource_manager_->import_component(std::move(gazeboSystem), control_hardware[i]);
   }
 
+  rclcpp_lifecycle::State state(
+    lifecycle_msgs::msg::State::PRIMARY_STATE_ACTIVE,
+    hardware_interface::lifecycle_state_names::ACTIVE);
+  resource_manager_->set_component_state("GazeboSimSystem", state);
```

Comment:
Isn't it supposed to be "GazeboSystem"? That's the name used in the [demos](https://github.com/ros-controls/gazebo_ros2_control/blob/0dbce66334b1b1e49dec923527e03897d8ffc74d/gazebo_ros2_control_demos/urdf/test_diff_drive.xacro.urdf#L135)

I

Reply:
Just saw that @samiamlabs [commented](https://github.com/ros-controls/gazebo_ros2_control/issues/131#issuecomment-1182865006) the same